test_cached_search_results failing in Actions

https://github.com/biglocalnews/WARN/runs/4808566629?check_suite_focus=true

```
=================================== FAILURES ===================================
__________________________ test_cached_search_results __________________________

tmp_path = PosixPath('/tmp/pytest-of-runner/pytest-0/test_cached_search_results0')

    @pytest.mark.vcr()
    def test_cached_search_results(tmp_path):
        "should save to cache when configured"
        url = "https://okjobmatch.com/search/warn_lookups"
        cache_dir = str(tmp_path.joinpath("ok"))
        site = JobCenterSite("OK", url, cache_dir=cache_dir)
        # The dates below should span two pages (just barely).
        # Skip detail pages to minimize fixture size.
        results_pages, data = site.scrape(
            start_date="2020-01-01",
            end_date="2020-04-15",
            detail_pages=False,
        )
        expected = ["2020-01-01_2020-04-15_page1.html", "2020-01-01_2020-04-15_page2.html"]
        files = [f.name for f in Path(cache_dir).glob("**/*.html")]
        assert len(files) == 2
>       assert files == expected
E       AssertionError: assert ['2020-01-01_...5_page1.html'] == ['2020-01-01_...5_page2.html']
E         At index 0 diff: '2020-01-01_2020-04-15_page2.html' != '2020-01-01_2020-04-15_page1.html'
E         Use -v to get the full diff

tests/test_job_center.py:116: AssertionError
=========================== short test summary info ============================
FAILED tests/test_job_center.py::test_cached_search_results - AssertionError:...
========================= 1 failed, 18 passed in 1.35s =========================
```

--- a/tests/test_job_center.py
+++ b/tests/test_job_center.py
@@ -113,7 +113,7 @@ def test_cached_search_results(tmp_path):
     expected = ["2020-01-01_2020-04-15_page1.html", "2020-01-01_2020-04-15_page2.html"]
     files = [f.name for f in Path(cache_dir).glob("**/*.html")]
     assert len(files) == 2
-    assert files == expected
+    assert sorted(files) == sorted(expected)
 
 
 @pytest.mark.vcr()

